packages/oscal-react-library/src/components/OSCALCatalogGroups.test.tsx

```diff
@@ -52,6 +52,73 @@ const testGroups: ControlGroup[] = [
   },
 ];
 
+const testGroupswithProps: ControlGroup[] = [
+  {
```

Comment:
Not sure how I feel about this test data here (including the test data before this one). Maybe this could be moved within `test-data`?

Reply:
That's fine. I really dislike `test-data` but it exists and there's not much to do about that. If you move it, keep it in a TSX file.

Reply:
I think I'll just plan to leave it where it is. Would it make sense to move more of our data to the file where we are testing it in the future?

Reply:
I personally think having a huge list of "common" objects that we continuously add to makes testing a bit tricky. I think that testing the smallest unit possible with very minimum data makes our tests more understandable and makes it easier to test various edge cases. 

At the very least, we need to stop the pattern of missing data for tests, adding it to the already huge objects in `test-data` and making our tests more complex.